### PR TITLE
Feature/UI updates - Reorganizing Chatflow Dashboard

### PR DESCRIPTION
**Pull request by vinodkiran**

### CODE REVIEW

**Comment by vinodkiran** [comment]
image
image
image
image
image

**Comment by HenryHengZJ** [comment]
When minimized, scroll table to right, no other columns are displayed except just the Name: 
image

**Comment by HenryHengZJ** [comment]
In dark mode, we need to update icons and wordings accordingly as well: 
image

**Comment by HenryHengZJ** [comment]
Category doesnt seems to work. I tried update category but nothing comes back, database is not updated too: 
image

File: packages/ui/src/ui-component/table/FlowListTable.js [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+                                                <Typography
+                                                    sx={{ alignItems: 'center', display: 'flex', fontSize: '.8rem', fontWeight: 200 }}
+                                                >
+                                                    + {images[row.id].length - 5} More
+                                                </Typography>
+                                            )}
+                                        </div>
+                                    )}
+                                </TableCell>
+                                <TableCell sx={{ display: { xs: 'none', sm: 'table-cell' } }} key='3'>
```
Comment (HenryHengZJ) [inline packages/ui/src/ui-component/table/FlowListTable.js:132]: why do we hide the column when `xs` ?

**Comment by vinodkiran** [comment]
> Category doesnt seems to work. I tried update category but nothing comes back, database is not updated too: 
> 
> image
> 
> 

Interesting! Hit an enter after you type in a category. That should add a tag (displayed above the text field). I will fix this bug that you have found.

**Comment by vinodkiran** [comment]
> When minimized, scroll table to right, no other columns are displayed except just the Name: 
> 
> image
> 
> 

This was intentional - columns are set to hide for small screens (sm). Should we show all fields for "sm" layouts? Menu ?

**Comment by vinodkiran** [comment]
All additions/updates are done, and the PR can be considered complete.

**Comment by HenryHengZJ** [comment]
> > When minimized, scroll table to right, no other columns are displayed except just the Name:
> > image
> 
> This was intentional - columns are set to hide for small screens (xs). Should we show all fields for "xs" layouts? What about the Menu for "xs" layouts?

I think we should show it as well, similar to other tables in Credentials, API Keys

**Review by HenryHengZJ — APPROVED** [review]

**Comment by HenryHengZJ** [comment]
@vinodkiran let me know if you have any outstanding questions/issues before merging

**Comment by vinodkiran** [comment]
> @vinodkiran let me know if you have any outstanding questions/issues before merging

Nope, please go ahead with the merge.

**Comment by HenryHengZJ** [comment]
awesome thanks @vinodkiran !

**Pull request merged**